### PR TITLE
access token verification

### PR DESCRIPTION
**Pull request by ridhishjain**

### CODE REVIEW

File: server/models/auth/getUser.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const { pool } = require('../database')
 function getUser(username) {
   return new Promise((resolve, reject) => {
     pool.query(
-      `SELECT username,full_name,email,admission_number,mobile,department,branch,bio,logged_in FROM user WHERE username=?`,
+      `SELECT username,full_name,email,admission_number,mobile,department,branch,bio,logged_in,profile_img FROM user WHERE username=?`,
```
Comment (ridhishjain) [inline server/models/auth/getUser.js:12]: profile image link also sending in getUser API

File: server/routes/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ router.use('/auth', authRouter.updatePasswordRouter)
 router.use('/auth', authRouter.updateUserRouter)
 router.use('/auth', authRouter.uploadProfileImage)
 router.use('/auth', authRouter.verifyNewEmailRouter)
-router.use('/users', authRouter.getUserRouter)
+router.use('/user', authRouter.getUserRouter)
```
Comment (ridhishjain) [inline server/routes/index.js:24]: makes more sense

**Comment by ridhishjain** [comment]
**All the APIs are working fine!!**
**Docs - [here](https://documenter.getpostman.com/view/7963087/TVKJwEH1#c0b7656b-1692-44d2-848b-ee119ec62b52)**

File: server/models/auth/getUser.js (hunk 1 above)
Comment (cjchirag7) [reply server/models/auth/getUser.js:12]: great !

File: server/routes/index.js (hunk 2 above)
Comment (cjchirag7) [reply server/routes/index.js:24]: cool

**Review by cjchirag7 — APPROVED** [review]
LGTM !

**Pull request merged**